templates/base.html.twig

```diff
@@ -33,45 +33,50 @@
 								</li>
 							{% endif %}
 						{% endif %}
-					{% if not app.user %}
-						{% if not is_granted("ROLE_ADOPTER") or not is_granted("ROLE_ANNOUNCER") or not is_granted("ROLE_ADMIN") %}
+						{% if not app.user %}
+							{% if not is_granted("ROLE_ADOPTER") or not is_granted("ROLE_ANNOUNCER") or not is_granted("ROLE_ADMIN") %}
```

Comment:
Je pense que tu peux enlever cette ligne car il y a déjà le "IF NOT APP.USER" (si personne n'est connecté) 
-> Affiche inscription